test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -1968,6 +1968,25 @@ def mkdtemp(self, hostname=None, suffix='', prefix='PtlPbs', dir=None,
                        sudo=True)
         return fn
 
+    def create_dyn_res_script(self, body, prefix=None, suffix=None, perm=0755,
+                              dirname=None, host=None):
+        """
+        Create a dynamic resource script owned by root
+        """
+        home_dir = os.path.expanduser("~")
```

Comment:
if this file is supposed to be owned by root, then shouldn't you create it in root's home directory instead of the current user's ?

Reply:
Since while testing pbs_benchpress runs as pbsroot I don't think this user will have privileges to create a file in root's home directory. This is why I create it in user's home and copy it as root to temp directory (or any other specified directory)